1048/en.md

```diff
@@ -0,0 +1,146 @@
+## Problem Restatement. 
+
+We Have an Array ```a``` of ```n+1``` elements, and in that we want to mark ```k``` points. And the output of this problem is the function
+
+```
+f(a,v) = max(v[i + 1] - v[i]) over all i in set of marked indices.
+```
```

Comment:
What is `v`? Please define.
Personally I feel you can say, "Given an array, we have to partition it into `k+1` sub-arrays such that the maximum sub-array sum is minimized".